components/ws-daemon/pkg/content/initializer.go

```diff
@@ -399,6 +399,9 @@ func (rs *remoteContentStorage) Download(ctx context.Context, destination string
 
 	args := []string{
 		"-x16", "-j12",
+		"--retry-wait=5",
```

Comment:
Good idea.

For posterity, from [options](https://aria2.github.io/manual/en/html/aria2c.html#options):
```
--retry-wait=<SEC>[¶](https://aria2.github.io/manual/en/html/aria2c.html#cmdoption-retry-wait)
Set the seconds to wait between retries. When SEC > 0, aria2 will retry downloads when the HTTP server returns a 503 response. Default: 0
```

Reply:
:eyes: